R/pkg/R/mllib.R

```diff
@@ -197,11 +197,10 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
-#'
 #' Makes predictions from a generalized linear model produced by glm() or spark.glm(),
 #' similarly to R's predict().
 #'
+#' @title predict
```

Comment:
Shouldn't we follow a single convention for defining the title, either using ```@title``` or using the first sentence in the description?  @shivaram do you have a preference?  Looking at the Github history, I see lots of SparkR contributors do both.